src/score_hv/harvesters/global_bucket_precip_ave.py

```diff
@@ -0,0 +1,129 @@
+import os,sys
+
+from collections import namedtuple
+from dataclasses import dataclass
+from dataclasses import field
+import numpy as np
+import netCDF4 as nc 
```

Comment:
(line 7): nothing wrong with this, but I usually use "from netCDF4 import Dataset" which is a bit more specific. Whichever we use, I think we should be consistent moving forward.